spec/unit/lib/temporal/testing_spec.rb

```diff
@@ -30,6 +37,65 @@ def execute; end
     end
   end
 
+  describe 'In local testing mode, Temporal.schedule_workflow' do
+    around do |example|
+      Temporal::Testing.local! { example.run }
+    end
+
+    it 'allows the test to simulate deferred executions' do
+      workflow = TestTemporalOverrideWorkflow.new(nil)
+      allow(TestTemporalOverrideWorkflow).to receive(:new).and_return(workflow)
+      workflow2 = TestTemporalOverride2Workflow.new(nil)
+      allow(TestTemporalOverride2Workflow).to receive(:new).and_return(workflow2)
```

Comment:
You can simply:

```ruby
workflow = TestTemporalOverrideWorkflow.new(nil)
workflow2 = TestTemporalOverrideWorkflow.new(nil)

allow(TestTemporalOverrideWorkflow).to receive(:new).and_return(workflow, workflow2)
```

avoiding the need for a second blank workflow, which doesn't really do anything